src/PartielsPy/Partiels.py

```diff
@@ -0,0 +1,49 @@
+"""A main class for Partiels Wrapper"""
+
+import os
+import platform
+import shutil
+
+
+class Partiels:
+    def __init__(self):
+        self.__executable_path = self.__findExecutablePath()
+
+    def __findExecutablePath(self):
+        name = "Partiels"
+        system = platform.system()
+        path = os.environ.get("PARTIELS_PATH")
+        if path:
+            return path
+        path = shutil.which(name)
+        if path:
+            return path
+        if system == "Linux":
+            paths = [
+                os.path.join(os.environ.get("HOME"), "opt", name),
+                os.path.join("/opt/", name),
+            ]
+        elif system == "Windows":
+            architecture = platform.architecture()[0][:2]
+            paths = [
+                os.path.join("autopf" + architecture, name + ".exe"),
+                os.path.join(
+                    os.environ["ProgramFiles"],
+                    name,
+                    name + ".exe",
+                ),
+            ]
+        elif system == "Darwin":
+            paths = [
+                os.path.join("/Applications", name + ".app", "Contents", "MacOS", name),
+            ]
+        for path in paths:
+            if os.path.isfile(path) and os.access(path, os.X_OK):
```

Comment:
There is no need for path to contain the name of the executable if you use `shutil.which()` because this will be managed by the function depending on the os.